sjb/generate.py

```diff
@@ -108,7 +108,7 @@ def load_configuration(config_path):
     # now, the job can define actions to take
     for action in job_config.get("actions", []):
         if action["type"] == "script":
-            actions.append(ScriptAction(action.get("repository", None), action["script"], action.get("title", None), action.get("timeout", 7200)))
+            actions.append(ScriptAction(action.get("repository", None), action["script"], action.get("title", None), action.get("timeout", 14400)))
```

Comment:
(you had me write the ScriptAction init as: `def __init__(self, repository, script, title, timeout=14400):`  so now you can't pass None if you want that default to do anything).